src/domains/wallet/pages/ImportWallet/Ledger/LedgerAuthStep.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useLayoutEffect, useState } from "react";
+
+import { useTranslation } from "react-i18next";
+import { Image } from "@/app/components/Image";
+import { Header } from "@/app/components/Header";
+import { Spinner } from "@/app/components/Spinner";
+import { useLedgerContext } from "@/app/contexts";
+
+export const LedgerAuthStep = ({
+	onDeviceNotAvailable,
+	onDeviceAvailable,
+}: {
+	onDeviceNotAvailable: () => void;
+	onDeviceAvailable: () => void;
+}) => {
+	const { t } = useTranslation();
+	const [isLookingDevice, setIsLookingDevice] = useState(true);
+	const [isLedgerConnected, setIsLedgerConnected] = useState(false);
+
+	const { listenDevice, hasDeviceAvailable, isAwaitingConnection } = useLedgerContext();
+
+	useLayoutEffect(() => {
+		(async () => {
+			await listenDevice();
+			setIsLookingDevice(false);
+		})();
+	}, [setIsLookingDevice, listenDevice]);
+
+	useLayoutEffect(() => {
+		if (hasDeviceAvailable && !isLookingDevice) {
+			setIsLedgerConnected(true);
+
+			onDeviceAvailable();
+		}
+	}, [hasDeviceAvailable, isLookingDevice, onDeviceAvailable, setIsLedgerConnected]);
+
+	useLayoutEffect(() => {
+		if (!isAwaitingConnection && !hasDeviceAvailable && !isLookingDevice) {
+			setIsLedgerConnected(false);
+
+			onDeviceNotAvailable();
+		}
+	}, [isAwaitingConnection, hasDeviceAvailable, isLookingDevice, onDeviceNotAvailable, setIsLedgerConnected]);
+
+	return (
+		<section data-testid="LedgerAuthStep" className="space-y-8">
+			<Header title={t("WALLETS.CONNECT_LEDGER.HEADER")} />
+
+			<Image name="AuthLedgerDeviceDark" domain="wallet" className="mx-auto" useAccentColor={false} />
```

Comment:
@alfonsobries this should handle the light & dark colors of the image.

```suggestion
			<Image name="AuthLedgerDevice" domain="wallet" className="mx-auto" useAccentColor={false} />```